src/navigation/components/Standalone/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import RootCn from '../../../components/RootCn';
+
+import Navigation, {NavigationProps} from './../../components/Navigation/Navigation';
+
+const Standalone = (props: NavigationProps) => (
```

Comment:
Why is const exists and its name Standalone?

Reply:
I called it so because you can import and use this Navigation component regardless of constructor itself

Reply:
Can we add to readme the example of how to import navigation and use it with theme?